distributed/tests/test_versions.py

```diff
@@ -33,6 +33,11 @@ def kwargs_matching():
     )
 
 
+@pytest.fixture
+def column_matching():
+    return dict(scheduler=2, workers=3, client=1,)
```

Comment:
This doesn't need to be a fixture (it's not expensive to create, and is only used by one test. I suggest inlining in the test instead.